### PR TITLE
fix: tests

### PR DESCRIPTION
**Pull request by yannickrocks**
Resolves https://github.com/VorTECHsa/app-issues/issues/1060

### CODE REVIEW

File: tests/endpoints/test_cargo_movements_real.py [hunk 1]
```diff
@@ -187,7 +187,7 @@ def test_search_single_filter_owner_name(self):
             .to_df()
             .head(2)
         )
-        assert len(df) == 2
+        assert len(df) == 1
```
Comment (ChristianBorresen) [inline tests/endpoints/test_cargo_movements_real.py:190]: So we now have only one CM in that time frame for DHT, and previously we had two?

**Review by ChristianBorresen — CHANGES_REQUESTED** [review]

File: tests/endpoints/test_cargo_movements_real.py (hunk 1 above)
Comment (yannickrocks) [reply tests/endpoints/test_cargo_movements_real.py:190]: I believe so. I think the reason the data change might be when we changed the indices.

Comment (ChristianBorresen) [reply tests/endpoints/test_cargo_movements_real.py:190]: Cool. Terrible test though.

Comment (KitBurgess) [reply tests/endpoints/test_cargo_movements_real.py:190]: Terrible terrible

File: vortexasdk/version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "0.28.8"
+__version__ = "0.28.9"
```
Comment (RaduG) [inline vortexasdk/version.py:1]: Does this actually warrant a version bump? Tests are not really part of the package clients get.

**Review by ChristianBorresen — APPROVED** [review]

**Pull request merged**